[2024-08-04 00:14:53,247] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_taskflow_api_v02.get_age 2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04 00:14:53,385] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_taskflow_api_v02.get_age 2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04 00:14:53,403] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-08-04 00:14:53,404] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-08-04 00:14:53,406] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-08-04 00:14:53,489] {taskinstance.py:1063} INFO - Executing <Task(_PythonDecoratedOperator): get_age> on 2024-08-03T00:00:00+00:00
[2024-08-04 00:14:53,515] {standard_task_runner.py:52} INFO - Started process 34309 to run task
[2024-08-04 00:14:53,555] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_age', '2024-08-03T00:00:00+00:00', '--job-id', '12672', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpcyroar8g', '--error-file', '/tmp/tmp1hk25ksu']
[2024-08-04 00:14:53,590] {standard_task_runner.py:77} INFO - Job 12672: Subtask get_age
[2024-08-04 00:14:54,366] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_age 2024-08-03T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-08-04 00:14:55,108] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_taskflow_api_v02
AIRFLOW_CTX_TASK_ID=get_age
AIRFLOW_CTX_EXECUTION_DATE=2024-08-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-03T00:00:00+00:00
[2024-08-04 00:14:55,364] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_age, execution_date=20240803T000000, start_date=20240804T001453, end_date=20240804T001455
[2024-08-04 00:14:55,832] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-04 00:14:55,879] {local_task_job.py:146} INFO - Task exited with return code 0
